scriptTag/externalScriptTag.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE HTML>
+<!DOCTYPE html>
+<html>
+	<head>
+		<script type="text/javascript" src="script.css"></script>
```

Comment:
Please include **.js** file not **.css** file.

Reply:
done